index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <div class="main">
```

Comment:
Use main or section tag instead of a div to wrap the container

Reply:
Kindly try to use more friendly words like 'please' or 'kindly' to make your comment nice and friendly 😄 👍